test/monitoring/monitoring.go

```diff
@@ -75,9 +74,7 @@ func Cleanup(pid int) error {
 // PortForward sets up local port forward to the pod specified by the "app" label in the given namespace
 func PortForward(logf logging.FormatLogger, podList *v1.PodList, localPort, remotePort int, namespace string) (int, error) {
 	podName := podList.Items[0].Name
```

Comment:
FWIW, totes not your fault but because I've seen this now: This interface is quite awkward. Why pass a whole podList if all you want is just the name of the first pod anyway 🤔 .

Reply:
heh, very good question! Let's tidy that up separately, though